src/index.js

```diff
@@ -195,7 +195,26 @@ exports.init = (Implementation) => {
           'check on Forest that you copied it properly in the Forest ' +
           'initialization?');
       } else if (opts.envSecret) {
-        const collections = _.values(Schemas.schemas);
+        const forestadminSchemaFilename = `${path.resolve('.')}/forestadmin-schema.json`;
```

Comment:
I don't know if the liana is runnable on window, but if you do:
```
`${path.resolve('./forestadmin-schema.json')}`
```
it will correctly convert to `C:\...\forestadmin-schema.json`.

Reply:
It's the same as everywhere else in this file. If I change it here I should change them all to be consistent

Reply:
Ok